Revert new license syntax in order to drop setuptools requirement.

author: micketeer@gmail.com
--- pyproject.toml.orig	2025-06-18 08:30:02.796742494 +0000
+++ pyproject.toml	2025-06-18 08:30:58.900958579 +0000
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=77.0.0"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -7,11 +7,7 @@
 version = "0.22.1"
 description = "Python client for the Prometheus monitoring system."
 readme = "README.md"
-license = "Apache-2.0 AND BSD-2-Clause"
-license-files = [
-	"LICENSE",
-	"NOTICE",
-]
+license = { file = "LICENSE" }
 requires-python = ">=3.9"
 authors = [
     { name = "The Prometheus Authors", email = "prometheus-developers@googlegroups.com" },
